lessons/basics/composition.md

```diff
@@ -21,11 +21,11 @@ Modules are the best way to organize functions into a namespace.  In addition to
 Let's look at a basic example:
 
 ``` elixir
-defmodule Example do
-  def greeting(name) do
-    ~s(Hello #{name}.)
-  end
-end
+iex> defmodule Example do
```

Comment:
I think that It's not necessary use iex prompt here because the point is show the structure of `struct`, unlike when you want to show the **evaluation** of a piece of code.